presto-spi/src/main/java/com/facebook/presto/spi/type/VarcharType.java

```diff
@@ -29,6 +29,11 @@
     public static final int MAX_LENGTH = Integer.MAX_VALUE;
     public static final VarcharType VARCHAR = new VarcharType(MAX_LENGTH);
 
+    public static VarcharType createUnboundedVarcharType()
```

Comment:
This method was actually introduced by @arhimondr in another commit (not merged yet).
The fact that it does not create anything is not important - it is just implementation detail  - and I assumed that we want to get rid of the variable eventually, replacing it with more explicitly named type constructor.